### PR TITLE
Holder manual

### PR DESCRIPTION
**Pull request by irinale11**
created a command for manual control of the holder motor

### CODE REVIEW

File: src/main/java/frc/robot/commands/HolderMotorManual.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    // Use addRequirements() here to declare subsystem dependencies.
+    m_holder = subsystem;
+    addRequirements(m_holder);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    m_holder.runHolderMotor(CellevatorConstants.kHolderMotorSpeed); // runs the holder motor at a constant speed when the command starts 
+  }
```
Comment (PKuppili) [inline src/main/java/frc/robot/commands/HolderMotorManual.java]: This should be in execute()

**Review by PKuppili — CHANGES_REQUESTED** [review]
Please address the comment and fix the necessary code.

File: src/main/java/frc/robot/commands/HolderMotorManual.java (hunk 1 above)
Comment (irinale11) [reply src/main/java/frc/robot/commands/HolderMotorManual.java]: i moved the code to execute()

**Review by PKuppili — APPROVED** [review]
Looks good!

**Pull request merged**